src/background/background.html

```diff
@@ -8,6 +8,7 @@
 		<meta charset="utf-8">
 		<!-- async currently disabled, because of https://bugzilla.mozilla.org/show_bug.cgi?id=1506464 -->
 		<script type="application/javascript" src="../browser-polyfill.js"></script>
+		<script src="./modules/InstallUpgrade.js" type="module" charset="utf-8"></script>
```

Comment:
hmm why not just use ES6 modules as usual, i.e. `import` it in `background.js`?

Reply:
This was just copied from what you did in the Awesome Emoji Picker: https://github.com/rugk/awesome-emoji-picker/blob/a397f885592bdf3577b37c6d3d2f636921a0da4a/src/background/background.html#L11

Reply:
Ah yeah thanks, and the reason is given in here:

https://github.com/rugk/awesome-emoji-picker/blob/a397f885592bdf3577b37c6d3d2f636921a0da4a/src/background/modules/InstallUpgrade.js#L4-L5